src/app/clusters/resource-monitoring-server/resource-monitoring-cluster-objects.h

```diff
@@ -29,7 +29,9 @@ namespace app {
 namespace Clusters {
 namespace ResourceMonitoring {
 
-static constexpr std::array<ClusterId, 2> AliasedClusters = { HepaFilterMonitoring::Id, ActivatedCarbonFilterMonitoring::Id };
+// max of 20 characters + 1 for null terminator
```

Comment:
Why is there a null terminator involved?  There should be no null terminators involved.

Please document that the 20 comes from the constraint on ProductIdentifierValue in the spec.